### ISSUE TITLE
Notifications: Ability to modify included text

### ISSUE THREAD

**Issue by shlake** [IQSS/dataverse#2710]
Seems that the notifications (on account creation, dataset and/or dataverse creations and others) are "canned" and Harvard Dataverse specific. So for potential non-Harvard Dataverse installations, it would be nice to customize these notifications. I am glad to submit a github request for the following:

This request is the ability to customize the text of the auto notifications for a specific institution.

A few suggestions:
Account creation notification - has **support@dataverse.org** in the message. We would like to either be the **SystemEmail** (which is configurable), or to be able to change the text.

Dataverse & Dataset creation notification - has dataverse (or dataset) user guide link is included (Harvard's). We would like that to either be the **GuidesBaseUrl**, which can be configured, or to be able to edit the text.

BUT  since all the notification emails have "Thank you, The Dataverse Project". It looks like the best alternative would be to allow the text of the notifications be customized. UNLESS this closing can be configured via a setting.

**Comment by pdurbin** [comment]
@mheppler and I discussed this issue today when we were looking at pull request #3880 for issue #3880 and how `Bundle.properties` says `notification.email.welcome=Welcome to Dataverse! Get started by adding or finding data. Have questions? Check out the User Guide at {0}/{1}/user/ or contact Dataverse Support for assistance. Want to test out Dataverse features? Use our Demo Site at https://demo.dataverse.org`.

Probably instead of "Welcome to Dataverse!" it should say "Welcome to Libra Data!" or "Welcome to Scholars Portal!" or whatever the "root" dataverse name is.

Also, instead of "Dataverse Support", it should probably use the name from `:SystemEmail` such as "Libra Data" if it's configured as "Libra Data <support@dataverse.lib.virginia.edu>". See http://guides.dataverse.org/en/4.6.1/installation/config.html#systememail

Basically, there are two values that should be in play:

- The name of the installation, which I guess would be the name of the root dataverse.
- The name of the team who is doing support, which I would think would come from the `:SystemEmail` setting.

Incidentally, that message shouldn't mention the demo server at all. Perhaps part of that message should come from the database if we really need it for Harvard Dataverse.

**Comment by mheppler** [comment]
Briefly reviewed this issue with @mercecrosas and confirmed that we the "Root Dataverse" name is what is preferred to use as the welcome msg and support label branding name, as @pdurbin outlined above.

Working to outline the scope of these changes and hopefully include it as part of #3611 with the blessings of @djbrooke and @kcondon, since that issue is already making application-wide dataverse name changes.

**Comment by pdurbin** [comment]
@mheppler please let me know if https://github.com/IQSS/dataverse/blob/77214a5e48c12867b32ada820640a81b42291201/src/test/java/edu/harvard/iq/dataverse/branding/BrandingUtilTest.java accurately represents what we decided.

**Comment by pdurbin** [comment]
I just left a note about this issue at https://github.com/IQSS/dataverse/issues/3611#issuecomment-307133768 but I consider this scope of this issue to have complete control over the messaging, which we are not delivering in the other issue.

**Comment by mheppler** [comment]
With the customization features outline in #3639, #3611, #3637 as part of the 4.7 release, the request for customized branding in the notification has been delivered. The support email address and URL for the User Guide had been delivered in previous releases (as referenced in #3637). References to "Dataverse Support" and "Dataverse Project" have been made configurable in #3611.

I am going to close this issue, as delivered in 4.7 and previous releases. @shlake, if there are specific use cases we missed, please let us know and we can open this back up, or create a new issue.

**Comment by pdurbin** [comment]
I'd suggest following this new issue:

- #9220

See also this thread ("Change the text of the welcome email sent to new users"): https://groups.google.com/g/dataverse-community/c/n6DyufyA_7A/m/_REv-4SeAQAJ
